packages/compass-collection/src/components/workspace/workspace.spec.tsx

```diff
@@ -4,8 +4,9 @@ import { render, screen } from '@testing-library/react';
 import { spy } from 'sinon';
 import type AppRegistry from 'hadron-app-registry';
 
-import { Workspace, getTabType } from './workspace';
+import { Workspace } from './workspace';
 
+// TODO: why was this skipped?
```

Comment:
How long has this been skipped? Should we just remove it? What happens if you un-skip it?

Reply:
For awhile, I'm planning to re-enable them, doesn't look like they need to be skipped

Reply:
Re-enabled in https://github.com/mongodb-js/compass/pull/4858/commits/310a006462a24ba2a4f135a6b6efa5d78022a9ce